src/main/java/clients/symphony/api/APIClient.java

```diff
@@ -19,7 +21,15 @@ protected void handleError(Response response, ISymClient botClient) throws SymCl
             );
             throw new ServerErrorException(response.getStatusInfo().getReasonPhrase());
         } else {
-            ClientError error = response.readEntity(ClientError.class);
+            String errorString = response.readEntity(String.class);
```

Comment:
Could we try to set a Content-Type header to application/json if it does not exist ?

Or at least extract the parsing into a dedicated function. And put the ObjectMapper as static field since it is quite a heavy object designed for reuse, see documentation : https://fasterxml.github.io/jackson-databind/javadoc/2.7/com/fasterxml/jackson/databind/ObjectMapper.html#ObjectMapper()

Reply:
I tried to find way to set Content-Type header for response but I didn't find anything. Not sure if it's possible or not.
Maybe I will try the second suggestion.